project/FrgaalJavaCompiler.scala

```diff
@@ -321,7 +321,7 @@ object FrgaalJavaCompiler {
       )
     val allArguments = outputOption ++ frgaalOptions ++ nonJArgs ++ allSources
 
-    if (Integer.parseInt(target) >= 24) {
+    if (false) { // disable using JDK's `javac` for now
```

Comment:
- `jvm-channel` is using `java.foreign.memory` API not available in JDK 21 (without a preview)
- Frgaal 21 had problems understanding JDK 24 bytecode
- we had to use `javac` when the target was higher than 21
- https://github.com/enso-org/enso/pull/13076#discussion_r2134442767
- #13244
- with Frgaal 24, there is no need for that